stock_reception_screen_mrp_subcontracting/wizards/mrp_product_produce.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class MrpProductProduce(models.TransientModel):
+    _inherit = "mrp.product.produce"
+
+    def _record_production(self):
+        # Overridden to unlink existing move lines when the components
+        # consumption is recorded. As a result we will get only new move lines
+        # related to the production, avoiding issues when processing them with
+        # the reception screen.
```

Comment:
shall we run this only when ctx key from reception screen is there?

Reply:
The context key is not enough (and doesn't make sense too) but instead run this only when the destination move is related to a reception transfer, because we usually call this wizard before using the reception screen (and this add move lines on the reception transfer difficult to handle with the reception screen...).